app/bundles/LeadBundle/Helper/IdentifyCompanyHelper.php

```diff
@@ -49,6 +48,8 @@ public static function identifyLeadsCompany($parameters, Lead $lead, CompanyMode
             $companyName = filter_var($parameters['company']);
         } elseif (isset($parameters['email'])) {
             $companyName = $companyDomain = self::domainExists($parameters['email']);
+        } elseif (isset($parameters['companyname'])) {
+            $companyName = filter_var($parameters['companyname'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
```

Comment:
Will this result in the same issue as https://github.com/mautic/mautic/pull/2867?